### PR TITLE
mostly worky gpt3

### PR DESCRIPTION
**Pull request by supertick**

### CODE REVIEW

File: src/main/java/org/myrobotlab/service/config/Gpt3Config.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package org.myrobotlab.service.config;
+
+public class Gpt3Config extends ServiceConfig {
+
+  public String currentBotName = "Alice";
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/config/Gpt3Config.java]: Why alice?

Comment (supertick) [reply src/main/java/org/myrobotlab/service/config/Gpt3Config.java]: There are probably vars and behaviors that probably should work the same way between ProgramAB and ChatGpt3.  
I guess this is a reminder to start heading in that direction.  Heh, a little experimentation came up with a non-personalized response. 
image

Comment (supertick) [reply src/main/java/org/myrobotlab/service/config/Gpt3Config.java]: removing it for now .. but it would be nice to name it what you want ...

Comment (kwatters) [reply src/main/java/org/myrobotlab/service/config/Gpt3Config.java]: Alice refers to a specific AIML set.  IF we want to give this one a name.. it should simply be GPT-3...

File: src/main/java/org/myrobotlab/service/Gpt3.java [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+  public Gpt3(String n, String id) {
+    super(n, id);
+  }
+
+  public Response getResponse(String text) {
+
+    try {
+
+      Gpt3Config c = (Gpt3Config) config;
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/Gpt3.java:72]: I don't like that this needs to be type casted here on each response.

Comment (supertick) [reply src/main/java/org/myrobotlab/service/Gpt3.java:72]: The framework sets the value ... all the services don't need to worry about handling the config
the getConfig/apply is handled completely by the framework.
To do so its a single field in the Service.java definition is needed.   If not this,
what would you suggest ?

File: src/main/java/org/myrobotlab/service/config/Gpt3Config.java (hunk 1 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/service/config/Gpt3Config.java]: I just removed it for now ... there are some meaningful features  (sleep, wake, bot name, session name, user name)
which would be useful to implement - but this is only the initial implementation

**Pull request merged**